### PR TITLE
Simplify code in Jinja templates

### PR DESCRIPTION
**Pull request by kaxil**
We have lot of code duplication in our Jinja templates.

This PR simplifies it.

### CODE REVIEW

**Review by dimberman — APPROVED** [review]

**Review by feluelle — COMMENTED** [review]
I am not sure about this one. I like it to be explicit more - or to be fully automated i.e deserialisable. 

We also have created an issue to look into this. I can link it tomorrow morning (I am on the phone right now).

**Comment by kaxil** [comment]
> I am not sure about this one. I like it to be explicit more - or to be fully automated i.e deserialisable.
> 
> We also have created an issue to look into this. I can link it tomorrow morning (I am on the phone right now).

I feel this is explicit and reduces the surface area of errors when copy/pasting but I will leave it up to you to check tomorrow when you are back on laptop

File: sql-cli/sql_cli/macros/load_file.py.jinja2
Comment (pankajkoti) [inline sql-cli/sql_cli/macros/load_file.py.jinja2:12]: `path` is required param for `File`. So I believe we should have this outside the `if` so that we raise an error while generating the DAG if the user has not specified `path`.

**Review by pankajkoti — COMMENTED** [review]
Looks good to me, but have a suggestion.

**Review by pankajkoti — APPROVED** [review]
Looks clean to me, thanks !

**Review by feluelle — CHANGES_REQUESTED** [review]

File: sql-cli/sql_cli/macros/load_file.py.jinja2
Comment (kaxil) [reply sql-cli/sql_cli/macros/load_file.py.jinja2:40]: oh good point, but I think they were not being handled even without the PR

Comment (feluelle) [reply sql-cli/sql_cli/macros/load_file.py.jinja2:40]: Or a new example maybe

**Comment by tatiana** [comment]
> I am not sure about this one. I like it to be explicit more - or to be fully automated i.e deserialisable.
> 
> We also have created an issue to look into this. I can link it tomorrow morning (I am on the phone right now).

@feluelle The ticket #1284 is related to this PR, right?
https://github.com/astronomer/astro-sdk/issues/

I remember us talking about this and that there are possibly other approaches you'd like to try out - including what @dimberman did with the Databricks implementation.

I think we have two ways of moving forward:
1) We merge this change, and we revisit it after - as part of #1284 (if we think it has more pros than the current implementation)
2) We don't merge this change and review the approach as part o #1284 (I'm adding it to the milestone 0.4)

I'm happy for @kaxil and @feluelle to make a call on this - or we can discuss as a team in the next session if we don't converge before :)

**Comment by feluelle** [comment]
I am in favor of 2 - not merging and reviewing it as part of ticket 1284.

In the current state it would even break existing functionality, see my https://github.com/astronomer/astro-sdk/pull/1367#discussion_r1041064284. So we would have to be more careful on the type per loop. We basically need a loop for strings & raw values such as booleans. That's why I am in favor of being explicit here.

File: sql-cli/sql_cli/macros/load_file.py.jinja2
Comment (kaxil) [reply sql-cli/sql_cli/macros/load_file.py.jinja2:40]: Fixed, tests not added

**Comment by kaxil** [comment]
>In the current state it would even break existing functionality, see my https://github.com/astronomer/astro-sdk/pull/1367#discussion_r1041064284.

Fixed the breaking change :) but no strong opinion

**Comment by kaxil** [comment]
Closing it for now, you can always re-open it if needed when someone picks up that task

**Pull request closed without merging**
